examples/ex_connectivity_tutorial/ex_connectivity_tutorial.pro

```diff
@@ -1,6 +1,6 @@
 #--------------------------------------------------------------------------------------------------------------
 #
-# @file     connectivityTutorial.pro
+# @file     ex_connectivity_tutorial.pro
```

Comment:
do we want to have tutorial in here? example or tutorial we should decide for one. I like ex_ a bit better.